src/storage-blob-preview/azext_storage_blob_preview/operations/blob.py

```diff
@@ -618,11 +634,13 @@ def upload_blob(cmd, client, file_path=None, container_name=None, blob_name=None
     try:
         if file_path:
             length = os.path.getsize(file_path)
+            _adjust_block_blob_size(client, blob_type, length)
```

Comment:
this should only applied to block blob, not page or append blob. please add check.

Reply:
It's checked in `_adjust_block_blob_size`:
```Python
def _adjust_block_blob_size(client, blob_type, length):
    if not blob_type or blob_type != 'block':
        return
```